portality/scripts/inconsistent_journal_prov_application.py

```diff
@@ -174,6 +180,6 @@ def journals_applications_provenance(outfile_applications, outfile_accounts, out
 
 if __name__ == "__main__":
     print 'Starting {0}.'.format(datetime.now())
-    applications_inconsistencies("apps_with_prov.csv", "apps_accepted_without_journals.csv", local)
+    # applications_inconsistencies("apps_with_prov.csv", "apps_accepted_without_journals.csv", local)
```

Comment:
Did you mean to leave this report commented out?

Reply:
Gah, no, I didn't sorry, I was testing just that one bit and forgot to uncomment